modules/EnsEMBL/Web/Configuration/Gene.pm

```diff
@@ -52,8 +52,14 @@ sub modify_tree {
     { 'availability' => $self->has_insects_gene_tree($clusterset_ids) }
   );
 
+  my $drosophilidae_node = $self->create_node('Drosophilidae_Tree', 'Gene tree (Drosophilidae)',
+    [qw( image EnsEMBL::Web::Component::Gene::DrosophilidaeTree )],
+    { 'availability' => $self->has_drosophilidae_gene_tree($clusterset_ids) }
+  );
+
   $genetree_menu->after($protostomes_node);
   $protostomes_node -> after($insects_node);
+  $insects_node -> after($drosophilidae_node);
```

Comment:
I wonder whether we should hide this node for any species that aren't Drosophilas. With the above line, all non-drosophilas will be showing a grey disabled link.

Reply:
This can indeed be done. Here is what Apis mellifera looks like with the Drosophila pangenome gene-tree link removed: http://wp-np2-25.ebi.ac.uk:5094/Apis_mellifera/Gene/Compara_Tree?g=LOC726692

If this is what you have in mind, I could push a commit with this optional Drosophilidae link.

Reply:
Personally, if I were a user, I would rather not see a disabled link to the Drosophilidae tree on every organism that isn't Drosophila, like what you have in your sandbox. But ultimately, it's your (or the Metazoa team's) call.

@jgtate ? Advice?